setup.cfg

```diff
@@ -51,6 +51,7 @@ zip_safe = False
 install_requires =
     defusedxml
     packaging
+    Pillow>=9.5.0
```

Comment:
Could we loosen this requirement?

Ideally we want a minimum version, and then if we can add an entry to our `constraints.txt` with this version, that would be perfect.

Reply:
@adehad done, I set the requirement to the minimum version required for this to work (2.1.0). Thanks!

Reply:
Awesome, made a small correction to the `constraints.txt` - we use a `==` locked/pinned version in that file.

Will merge once the CI is happy, thanks again!